lib/jmespath/nodes/function.rb

```diff
@@ -145,8 +152,10 @@ def call(args)
         if args.count == 2
           haystack = args[0]
           needle = args[1]
-          if String === haystack || Array === haystack
-            haystack.include?(needle)
+          if haystack.respond_to?(:to_str)
+            haystack.to_str.include?(needle)
```

Comment:
need to be more consistent in updating String/#to_str - I do here but not other places